test/cpu_fusion.cpp

```diff
@@ -143,7 +143,7 @@ TEST(cpu_fusion, gemm_cpu_broadcast_row)
 
     auto f = make_shared<Function>(cg, op::ParameterVector{A, B});
 
-    auto backend = runtime::Backend::create("CPU");
+    shared_ptr<runtime::Backend> backend = runtime::Backend::create("CPU");
```

Comment:
why `shared_ptr` used here instead `uniq_ptr`?

Reply:
I got lazy. Some of the tests depend on backend being a shared_ptr so I just did a global search/replace.